modules/executors/main.tf

```diff
@@ -73,6 +73,7 @@ resource "aws_security_group" "metrics_access" {
 
 resource "aws_cloudwatch_log_group" "syslogs" {
   # TODO: This is hardcoded in the executor image.
+  count             = var.need_syslogs ? 1 : 0
```

Comment:
ideally i think we'd pull some of this stuff out into a different module altogether

Reply:
I was wondering if using a data source to check for existence might work. We don't actually need to create it, that happens automatically when the first log comes in, we just want to configure retention, really.